src/xref2/link.ml

```diff
@@ -403,13 +400,74 @@ and open_ env parent = function
 let rec unit env t =
   let open Compilation_unit in
   let content =
-    match t.content with
-    | Module sg ->
-        let sg = signature env (t.id :> Id.Signature.t) sg in
-        Module sg
-    | Pack _ as p -> p
+    if t.hidden then t.content
+    else
+      match t.content with
+      | Module sg ->
+          let sg = signature env (t.id :> Id.Signature.t) sg in
+          Module sg
+      | Pack _ as p -> p
+  in
+  let source_info =
+    let open Source_info in
+    match t.source_info with
+    | Some inf ->
+        let jump_to v f_impl f_doc =
+          let documentation =
+            match v.documentation with Some p -> Some (f_doc p) | None -> None
```

Comment:
As documentation link are not used, would it make sense to comment out this code ? It would be a shame to loose it though.

Reply:
"Links to documentation" are used to count occurrences, I propose to keep that feature, otherwise I need to rename the PR ^^.
The name is wrong though. Maybe `type jump_to = {documentation : ... ; implementation : ...}` could be renamed to `type occurrence_of = ...`? (and this function would also be renamed)

Reply:
Right! I focused on the html generator and noticed that it wasn't used there and wrongly assumed it wasn't used at all.
Perhaps the code in `document` could be commented-out ?

Reply:
I commented the code in document extracting an href to documentation from an occurrence.